src/handlers.ts

```diff
@@ -4,21 +4,63 @@ import {
     exceptions,
     handlerEvent,
     HandlerErrorCode,
+    integer,
+    Integer,
     LoggerProxy,
     OperationStatus,
     Optional,
     ProgressEvent,
     ResourceHandlerRequest,
     SessionProxy
 } from 'cfn-rpdk';
+import fetch, { Response } from 'node-fetch';
 
 import { ResourceModel } from './models';
 
 interface CallbackContext extends Record<string, any> {};
 
-const MONITORING_PAGE = 'https://status.aws.amazon.com';
+interface Monitor {
+    readonly id?: string;
+    readonly name?: string;
+    readonly uri?: string;
+    readonly type?: string;
+    readonly frequency?: integer;
+    readonly status?: string;
+    readonly locations?: string[];
+    readonly slaThreshold?: number;
+};
 
 class Resource extends BaseResource<ResourceModel> {
+    static readonly API_ENDPOINT = 'https://synthetics.eu.newrelic.com/synthetics/api';
```

Comment:
Add the US endpoint too.  Add to schema as a location enum (user picks US or EU, assign appropriate endpoint here)?

Reply:
Done